dist/src/Form/Extension/ProductTypeExtension.php

```diff
@@ -26,6 +26,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         $builder->add('description', RichEditorType::class, [
             'required' => false,
             'label' => 'sylius.form.product.description',
+            'locale' => $builder->getName(),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Why `getName` here ?

Reply:
yes, the current name contains the local code (cf `ResourceTranslationsType` class)